src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -61,5 +63,12 @@ public function prepend(ContainerBuilder $container)
                 'client_manager' => 'sylius.oauth_server.client_manager',
             ],
         ]);
+
+        $container->setDefinition('fos_oauth_server.controller.token', new Definition(
+            'Sylius\Bundle\ApiBundle\Controller\TokenController', [
```

Comment:
Anyway, something's wrong with the indentation of this definition, it looks weird.